sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_sanitized_v3/query.sql

```diff
@@ -34,6 +34,34 @@ WITH impressions AS (
     `moz-fx-data-shared-prod.contextual_services_stable.quicksuggest_impression_v1`
   WHERE
     DATE(submission_timestamp) = @submission_date
+
+  UNION ALL
+
+  SELECT
+    TIMESTAMP_TRUNC(submission_timestamp, SECOND) AS submission_timestamp,
+    metrics.string.quick_suggest_request_id AS request_id,
+    NULL AS telemetry_query,
+    metrics.string.quick_suggest_advertiser AS advertiser,
+    CAST(metrics.string.quick_suggest_block_id AS INT64) AS block_id,
+    metrics.uuid.quick_suggest_context_id AS context_id,
+    sample_id,
+    metrics.boolean.quick_suggest_is_clicked AS is_clicked,
+    client_info.locale AS locale,
+    metadata.geo.country,
+    metadata.geo.subdivision1 AS region,
+    normalized_os,
+    normalized_os_version,
+    client_info.app_channel AS normalized_channel,
```

Comment:
i'm pretty sure `normalized_channel` is a top level column on the table

Reply:
Good catch, I missed that. Fixed.